cmd/anonymizer/main.go

```diff
@@ -53,7 +54,7 @@ func main() {
 				},
 			}
 
-			writer, err := writer.New(conf, logger)
+			writerObj, err := writer.New(conf, logger)
```

Comment:
if you don't like the name shadowing, then use `w`

Reply:
Made the changes.